### ISSUE TITLE
matplotlib v2 deprecation warnings

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#477]
When I start Cuts plugin and have matplotlib v2 installed, I get the following warning:
```
2017-05-15 11:19:26,867 | W | cbook.py:136 (warn_deprecated) |
    .../matplotlib/cbook.py:136: MatplotlibDeprecationWarning:
    The axisbg attribute was deprecated in version 2.0. Use facecolor instead.
  warnings.warn(message, mplDeprecation, stacklevel=1)
```

When I use `Pick` and `Screenshot` (not sure which one issued it), I see this:
```
2017-05-15 14:29:02,864 | W | cbook.py:207 (deprecated_func) |
    .../ginga/util/plots.py:211: MatplotlibDeprecationWarning:
    The set_axis_bgcolor function was deprecated in version 2.0. Use set_facecolor instead.
  self.ax.set_axis_bgcolor('#303030')
```

**Comment by ejeschke** [comment]
@pllim, would fixing this for matplolib v2 be backward compatible with at least later 1.x versions?

**Comment by pllim** [comment]
I am not familiar enough with `matplotlib` development to answer for sure. There are some options:

* Use their suggestions, downgrade `matplotlib` to your min supported version (1.5.1), and see what happens.
* Ignore them until they are removed and cause error (probably too late by then).
* Put in `if ... else ...` to handle different `matplotlib` versions.
* Bump `matplotlib` requirement to 2.x and make non-backward compatible changes (probably too early to do this).

**Comment by pllim** [comment]
`matplotlib` 1.5.1 gives `AttributeError: Unknown property facecolor`, so we definitely need to handle the versions separately.

**Comment by pllim** [comment]
This should be addressed in #488.
